src/node/index.js

```diff
@@ -44,15 +48,29 @@ var lookupNumeric = function(list, code) {
   return false;
 };
 
-var lookupString = function(list, code) {
+let lookupString = function(list, code) {
   return list.indexOf(code) !== -1;
 };
 
-var _fetchDetails = function(code) {
-  throw Error("Not implemented yet");
+let _fetchDetails = function(code, cb) {
+  let url = BASE_URL + code;
```

Comment:
Make a function `createUrl` and use `createUrl(code)`, would be easier to update the logic in the future if used in multiple places.